app.yaml

```diff
@@ -1,8 +1,7 @@
-runtime: php72
+runtime: php73
 
 env_variables:
   ## Put production environment variables here.
-  APP_KEY: base64:8ivlD/xNPqI4KEkBTYWsBwbdklAQpRSt1S6V602PEGc=
```

Comment:
Is the one in production changed from this value?

Reply:
Yup, its different for each person's dev environment